src/components/VideoPreview.tsx

```diff
@@ -1,36 +1,78 @@
 import React from 'react'
 import { useVideo } from '@/api/hooks'
 import routes from '@/config/routes'
-import VideoPreviewBase, { VideoPreviewBaseMetaProps } from '@/shared/components/VideoPreviewBase/VideoPreviewBase'
+import VideoPreviewBase, {
+  VideoPreviewBaseMetaProps,
+  VideoPreviewBaseProps,
+  VideoPreviewPublisherProps,
+} from '@/shared/components/VideoPreviewBase/VideoPreviewBase'
+import { useDrafts } from '@/hooks'
+import { copyToClipboard } from '@/utils/broswer'
 
-type VideoPreviewProps = {
+export type VideoPreviewProps = {
   id?: string
-  progress?: number
-  className?: string
-} & VideoPreviewBaseMetaProps
+} & VideoPreviewBaseMetaProps &
+  Pick<VideoPreviewBaseProps, 'progress' | 'isLoading' | 'className'>
```

Comment:
I don't think we need the `isLoading` prop on those two components anymore, right? We can probably also remove it from the `useVideoSharedLogic`